packages/grafana-data/src/transformations/transformers/joinDataFrames.ts

```diff
@@ -38,6 +38,11 @@ export interface JoinOptions {
    */
   frames: DataFrame[];
 
+  /**
+   * @internal -- Optionally specify if the join should be an inner join
+   */
+  innerJoin?: boolean;
```

Comment:
```suggestion
  mode?: 'outer' | 'inner'; // defaults to 'outer'
```

This would let us have more options later without changing much

Reply:
Updated 👍